src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -30,7 +30,11 @@
       </v-col>
 
       <!-- Actions -->
-      <v-col cols="2" class="mt-n2" v-if="!hasNameTranslationChange && !isSummaryMode">
+      <v-col cols="2" class="mt-n2" v-if="
+      !hasNameTranslationChange &&
+      !isSummaryMode &&
+      !isLimitedConversionRestorationFiling &&
+      !isLimitedExtendRestorationFiling">
```

Comment:
I'm following an existing pattern, but this needs refactoring as it will become increasingly messy and difficult to read as the number of filings increase.  Also, it's impossible to add a new filing type without risking making inadvertent changes to functionality that's already working.

Reply:
I'd prefer to see a computed property for anything more than trivial logic -- such as this. As a bonus, you can document the getter to clarify what it's doing.

Reply:
That's a good idea.  I'll create a getter to replace that logic.